packages/pluggableWidgets/sidebar-web/src/components/Sidebar.tsx

```diff
@@ -18,7 +18,18 @@ export interface SidebarProps {
 }
 
 export function Sidebar(props: PropsWithChildren<SidebarProps>): ReactElement {
+    const sidebarRef = useRef<HTMLElement | null>(null);
     const expanded = useSidebar(!!props.startExpanded, props.name);
+    const CONTENT_SELECTOR = [
+        "mx-navigationtree",
+        "mx-navigationlist",
+        "mx-navbar",
+        "mx-menubar",
+        "mx-button",
+        "mx-link"
+    ]
+        .map(cls => "." + cls)
+        .join(",");
```

Comment:
Can do this outside of the function since it won't change inbetween renders? Then you can also remove the dependency in the `useEffect`

Reply:
Fixed